### PR TITLE
Added baner

### LINKED ISSUE #4: Added Baner
Created categories baner below nav bar for landing page

### PR DESCRIPTION
**Pull request by Raghav888**
Created categories baner below nav bar for landing page

### CODE REVIEW

File: src/components/categorybanner/CategoryBanner.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                <img className="book-type" src={item.categoryImage} alt="love-story" />
+                <div className="centered">{item.categoryName}</div>
+            </a>
+        </div>)
+})}
+    
+        
+    </div>
+    <div className="baner">
+        <a href="##">
```
Comment (JantuDeb) [inline src/components/categorybanner/CategoryBanner.js:28]: Replace these `<a >` with `<Link> ` from react router dom.

File: src/context/category-context.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const CategoryContext=createContext();
+
+const CategoryProvider=({children})=>{
+const [category,setCategory]=useState([]);
+
+useEffect(()=>{
+    (async () => {
+        try {
+          const response = await axios.get("/api/categories");
```
Comment (JantuDeb) [inline src/context/category-context.js:13]: You can destructure `data` from response 
```
const {data} = await axios.get(...)
```

File: src/context/category-context.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+          setCategory(response.data.categories);
+        } catch (error) {
+          console.log(error);
```
Comment (JantuDeb) [inline src/context/category-context.js:16]: You may create a component for error to show errors if theres any. For now it's good.

**Review by JantuDeb — COMMENTED** [review]
I have added some suggestions that you can take a look

**Review by JantuDeb — APPROVED** [review]

File: src/components/categorybanner/CategoryBanner.js (hunk 1 above)
Comment (Raghav888) [reply src/components/categorybanner/CategoryBanner.js:28]: Will do it once, those components are ready

File: src/context/category-context.js (hunk 2 above)
Comment (Raghav888) [reply src/context/category-context.js:13]: Okay got it

File: src/context/category-context.js (hunk 3 above)
Comment (Raghav888) [reply src/context/category-context.js:16]: yes will add in future

File: src/components/categorybanner/CategoryBanner.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    return(
+        <div>
+        <div className="product-line">
+        {
+category.map((item)=>{
+    return( 
+        <div key={item.id} className="display-image">
+            <a href="##">
+                <img className="book-type" src={item.categoryImage} alt="love-story" />
+                <div className="centered">{item.categoryName}</div>
```
Comment (tushartiwari7) [inline src/components/categorybanner/CategoryBanner.js:20]: use prettier.

File: src/pages/landing/home.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    html {
+        font-size: 50%;
+    }
+}
+
+@media only screen and (max-width: 640px) {
+    html {
+        font-size: 45%;
+    }
+}
```
Comment (tushartiwari7) [inline src/pages/landing/home.css:33]: It's awesome if your site becomes responsive with just this much media queries.

**Review by tushartiwari7 — APPROVED** [review]

**Pull request merged**